webpack.config.js

```diff
@@ -37,13 +37,14 @@ module.exports = (env, argv) => {
             noEmitOnErrors: true,
             minimizer: [
                 new ClosureWebpackPlugin({mode: 'STANDARD'}, {
-                    jscomp_error: ['accessControls', 'checkRegExp', 'checkTypes', 'checkVars',
+                    jscomp_error: ['accessControls', 'checkRegExp', 'checkVars', /*'checkTypes',*/
                                    'invalidCasts', 'missingProperties',
                                    'nonStandardJsDocs', 'strictModuleDepCheck', 'undefinedVars',
                                    'unknownDefines', 'visibility'],
-                    jscomp_off: ['fileoverviewTags', 'deprecated'],
+                    jscomp_off: ['fileoverviewTags', 'deprecated', 'uselessCode', 'suspiciousCode', 'checkTypes',],
```

Comment:
Yeah. See my comment about type checks. There are a lot of type errors on master that weren’t feasible to fix in this pr. 

Type errors:
7 - tonkenize.js
1 - parse_tables.js
1 - parser.js
1 - ast_nodes.js
55 - ast.js
14 - compile.js

I’m doing some learning on web pack because I currently have no real clue about it. At present jsbi and other libraries we use don’t pass type checking either. 

The reason jsbi wasn’t passing was because the node_module version is already minified- when I recently copied over the full version from their GitHub and let our compiler do the minifying and that seemed to work.